<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node name="display_detections" pkg="cnn_detect" type="display_detections" output="screen" >
      
      <rosparam file="$(find cnn_detect)/config/uav_detect.yaml" />
      <param name="uav_name" value="$(arg uav_name)" />

      <remap from="~camera_input" to="mobius/image_raw" />
      <remap from="~detections" to="cnn_detect/detections" />
    
    </node>
  
  </group>

</launch>
